invenio_drafts_resources/resource_unit.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Resource Unit."""
+
+
+class IdentifiedDraft:
```

Comment:
minor: this will probably need some changes to reflect who is it drafting. Probably treated in a following PR (I can do since Im touching database table etc.)

Reply:
Sure, I leave that up to you (you mean like keeping track of the `draft_of` object, right?)